app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,173 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$light-gray: #f2f2f2;
+$white: $white;
+$beach-blue: #5bc0de;
+
+body {
+  padding-top: 20px;
+}
+
+#index-background-header {
+  color: $white;
+  font-family: Circular,"Helvetica Neue",Helvetica,Arial,sans-serif;
+  font-size: 5em;
+  font-weight: bold;
+  padding-top: 300px;
+  text-align: center;
+}
+
+#search-bar-container .well {
+  background: rgba(0, 0, 0, 0.1);
+  border-color: transparent;
+  bottom: 0;
+  left: 0;
+  margin-bottom: 0;
+  position: absolute;
+  right: 0;
+}
+
+.columns {
+  padding-top: 150px;
+}
+
+.index-containers {
+  margin-top: 100px;
+  p {
+    color: inherit;
+  }
+  h1 {
+    color: inherit;
+  }
+  * {
+    color: $white;
+  }
+}
+
+
+
+#beach-snapshot {
+  span {
+    color: darken($light-gray, 30%);
+    padding-bottom: 100px;
+  }
+  h1 {
+    margin-bottom: 3px;
+  }
+  margin-bottom: 15px;
+}
+
+#beach-snapshot-icon-container {
+  margin-top: 25px;
+}
+
+.center {
+  text-align: center;
+}
+
+.inline-block {
+  display: inline-block;
+}
+
+.amenity-name {
+  vertical-align: 9px;
+}
+
+.col-centered {
+  display: inline-block;
+  float: none;
+  text-align: center;
+}
+
+.section-title {
+  display: inline-block;
+  vertical-align: 20px;
+}
+
+footer {
+  background: $beach-blue;
+  a, span {
```

Comment:
Rule declaration should be preceded by an empty line<br>Each selector in a comma sequence should be on its own line